plugins/inputs/ethtool/ethtool_linux.go

```diff
@@ -282,15 +282,15 @@ func (c *CommandEthtool) Interfaces(includeNamespaces bool) ([]NamespacedInterfa
 				Log:    c.Log,
 			}
 			if err := c.namespaceGoroutines[namespace].Start(); err != nil {
-				c.Log.Errorf(`Failed to start goroutine for namespace "%s": %s`, namespace, err)
+				c.Log.Errorf("Failed to start goroutine for namespace %q: %w", namespace, err)
```

Comment:
`%w` should be only used for creating error, not for printing. Use `%s` or `%v` instead.